labsystem/src/Icon.js

```diff
@@ -59,7 +61,9 @@ export default class Icon extends React.Component {
       "purple-60",
       "purple-70",
     ]),
+    /** Size of the rendered Icon. */
     size: PropTypes.oneOf(["small", "petit"]),
+    /** */
```

Comment:
```suggestion
    /** Add a class name to make custom changes */
```

we should've been using this more often, actually